src/platform_impl/linux/window.rs

```diff
@@ -101,17 +107,20 @@ impl Window {
       .insert(window_id);
 
     // Set Width/Height & Resizable
-    let win_scale_factor = window.scale_factor();
     let (width, height) = attributes
       .inner_size
       .map(|size| size.to_logical::<f64>(win_scale_factor as f64).into())
       .unwrap_or((800, 600));
-    window.set_default_size(1, 1);
```

Comment:
The reason this was removed is that it functions the same as `box_.set_size_request(min_width, min_height);`